### PR TITLE
v2.3.0

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues

* https://github.com/cityofaustin/atd-data-tech/issues/19532

---

#### Ship list

- [ ] Check migrations for any conflicts with latest migrations in `main` branch
- [ ] Confirm Hasura role permissions for necessary access
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

**Comment by johnclary** [comment]
sorry for confusion—this is the correct v2.3.0 release branch waiting for your approval 🙏

**Review by chiaberry — APPROVED** [review]
if you say its good to go, its good to go --

**Review by mddilley — APPROVED** [review]
🚀 🚢

**Pull request merged**
